packages/core/theme/src/controllers/xm-theme-controller.service.ts

```
@@ -83,19 +86,29 @@ export class XmThemeController implements OnDestroy {
 
+        this.styleManager.set('theme', this.getThemeHref(themeStrategy, file));
         this.activeTheme.next(theme);
     }
+
+    private getThemeHref(themeStrategy: ThemeStrategy, file: string): string {
+        const defaultPath = `/assets/themes/${file}.css`;
+        switch (themeStrategy) {
+            case 'TENANT_ONLY':
+                return `assets/css/${file}.css`;
+            case 'CUSTOM':
+                const {themePath} = this.publicConfig() || {};
```

Biome (2.5.5)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)